http/src/http.ml

```diff
@@ -14,56 +14,51 @@ module Transfer = struct
 end
 
 module Header = struct
-  module Private = struct
```

Comment:
I liked it better before, with the private module all separate, but I guess in this case that’s not an option

Reply:
Yes.